src/main/java/com/transportcompany/transportcompany/services/OrderServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.transportcompany.transportcompany.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.transportcompany.transportcompany.exceptions.OrderNotFoundException;
+import com.transportcompany.transportcompany.models.dtos.OrderDTO;
+import com.transportcompany.transportcompany.models.entities.Order;
+import com.transportcompany.transportcompany.repositories.OrderRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public  class OrderServiceImpl implements OrderService {
+
+    private final OrderRepository orderRepository;
+    private final ObjectMapper objectMapper;
+
+    public OrderServiceImpl(OrderRepository orderRepository, ObjectMapper objectMapper) {
```

Comment:
You can/should use @Autowired annotation on the constructor when you inject the dependencies.